### ISSUE TITLE
why is binary_crossentropy but not softmax_cross_entropy?

### ISSUE THREAD

**Issue by rcx986635** [fizyr/keras-retinanet#1006]
in losses.py line 58 , focal_loss choose keras.backend.binary_crossentropy(labels, classification)，why not softmax_cross_entropy  for multi-classification problems ?

thank you

**Comment by hgaiser** [comment]
We implemented the loss function based on the paper. There they say they use a sigmoid loss function. Sigmoid works fine for multi class classification too.
